pootle/settings/40-apps.conf

```diff
@@ -131,6 +131,23 @@ POOTLE_SIGNUP_ENABLED = True
 # chart with events
 POOTLE_REPORTS_MARK_FUNC = ''
 
+# Invoicing
+
+# Path to PhantomJS binary (needed to generate PDF invoices)
+POOTLE_INVOICES_PHANTOMJS_BIN = None
+
+# Base directory where PDF invoices will be created
+POOTLE_INVOICES_DIRECTORY = working_path('invoices')
```

Comment:
Won't this create problems if the directory is missing? I suggest creating the directory with a README file within to ensure the directory is always present, like we did for `dbs` directory: https://github.com/translate/pootle/tree/master/pootle/dbs